pkg/mkimage-raw-efi/make-raw

```diff
@@ -205,8 +205,14 @@ do_rootfs() {
         esac
     fi
 
-    # Copy rootfs to image A
-    dd if=$IMG of=$IMGFILE bs=1M conv=notrunc seek="$(( SEC_START * 512 ))" oflag=seek_bytes
+    if [ -n "$IMG" ]; then
+      # Copy rootfs
+      dd if="$IMG" of="$IMGFILE" bs=1M conv=notrunc seek="$(( SEC_START * 512 ))" oflag=seek_bytes
+    else
+      # clean first block of partition
+      # to avoid mounting of stale data
+      dd if=/dev/zero of="$IMGFILE" bs=512 count=1 conv=notrunc seek="$SEC_START"
```

Comment:
Why seek using sectors here and bytes on line 210? Is there a reason for them to differ?

Reply:
The main difference in block size. We use large block size on line 210 and should use bytes offset to seek and calculate it using `SEC_START * 512`. Here we use bs (block size) 512, and seek SEC_START * 512 in bytes equivalent with SEC_START * bs, so we can use seek in blocks and avoid unnecessary calculation.

Reply:
It might save 100us in CPU time, but it uses tens of seconds of human brain processing time each time anybody will read this code since they need to determine that it is actually writing to the same place.
I know which one I think is more expensive.

Reply:
Ha! This is exactly the opposite what I feel :) Because Petr already changed this line from what you, Erik, suggest here, to this current line, pointed by me. But anyway, this is very minor to me, both ways work.

Reply:
And for me was a surprise to see the special flag in dd 'seek_bytes', because when you often work with these commands you think in blocks and then you see bs=X and then seek=$BS * N, not noticing the 'seek_bytes', and then you brain stops here thinking 'hey there is an error, your block size is X, why you do this multiplication?"

Reply:
this is why I hanged for a couple of seconds on this particular line :)

Reply:
That's funny! One line, different view perspective, different path of thinking

Reply:
@rouming my only request is that the two dd lines use the same approach.